storymap/settings.py

```diff
@@ -0,0 +1,45 @@
+"""
```

Comment:
drop this and integrate instead with `core/loc.py` -- then instead of `import settings` use the environment variable.